### PR TITLE
MMCA-3963 : Non-compliance : Issue#4 - Two h1s

### PR DESCRIPTION
**Pull request by sameerhmrc**
JIRA Link : https://jira.tools.tax.service.gov.uk/browse/MMCA-3963

### CODE REVIEW

**Review by bsgangapat — APPROVED** [review]

File: app/views/AccountsView.scala.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
         errorSummary: components.errorSummary,
         button: components.button,
         checkboxes: components.inputCheckboxesWithLegend,
-        h1: components.h1,
+        h2: components.h2,
```
Comment (wg-hmrc) [inline app/views/AccountsView.scala.html:32]: @sameerhmrc please keep it as H1.. this file was fixed

**Review by wg-hmrc — CHANGES_REQUESTED** [review]
the change will cause accessibility issues

File: app/views/AccountsView.scala.html (hunk 1 above)
Comment (sameerhmrc) [reply app/views/AccountsView.scala.html]: @wg-hmrc this has now been reverted

**Comment by sameerhmrc** [comment]
> the change will cause accessibility issues

changes incorporated

**Review by wg-hmrc — APPROVED** [review]

**Pull request merged**